src/usb_moded-dyn-config.c

```diff
@@ -50,6 +50,10 @@ void list_item_free(mode_list_elem *list_item)
   free(list_item->android_extra_sysfs_value);
   free(list_item->android_extra_sysfs_path2);
   free(list_item->android_extra_sysfs_value2);
+  free(list_item->android_extra_sysfs_path3);
+  free(list_item->android_extra_sysfs_value3);
+  free(list_item->android_extra_sysfs_path4);
+  free(list_item->android_extra_sysfs_value4);
```

Comment:
If value is not set in config this will blowup as the android_extra_sysfs_value4 is not set to NULL/0, right?